### ISSUE TITLE
Send maximum amount fails in some cases

### ISSUE THREAD

**Issue by punkrock** [LiskHQ/lisk-desktop#4190]
If you hit “Send maximum amount”, and then entering the address or adding anything into the data field, the fee calculation will be wrong and fail.

**Comment by przemerr** [comment]
The fee is correct, but the amount stays the same, and since the fee goes up, the remaining amount is to low to pay for the fee. You need to re-click "send maximum amount" after every change. I'm not really sure if it's a bug, but definitely could cause some confusion.

**Comment by punkrock** [comment]
The sending process should be as smooth as possible. Not sure how to "fix" it as I guess they don't wanna automatically adjust the "amount"-field. Maybe force the users within the sending-popup to update the amount again by sending the "maximum amount"-button. It's tricky because Lisk Desktop doesn't really know what the users wants to do (sending max amount or just 0.1 LSK).

**Comment by reyraa** [comment]
Thanks @punkrock and @przemerr for bringing this topic up. We have discussed this at the time of designing and implementing this feature. 
As you said it too, we can not make decision on behalf of the user. You might click on Send Max Amount, then change your mind and send a little bit less. 
Or you might click on Send Max Amount and then reduce the size of the message, hence leaving some dust behind.

Important is, at the time of pressing "continue" to proceed to the summary page, the transaction should be valid, meaning, 
- The amount + fee must be less that your balance - minBalance.
- The fee should be enough to send the transaction.
Otherwise the "Continue" button must be disabled. 

If the app behaves as said, then it's working as expected. But if you can proceed to broadcast the transaction, then the network complain about insufficient fee, we should fix it.

**Comment by przemerr** [comment]
yes, you get the minBalance error, since the amount is to big after a text field change. 

To reproduce, just try to send a TX, use the "send max amount", type something in the textfield and go to confirmation.
![image](https://user-images.githubusercontent.com/25085095/156568884-cd49d758-2b5e-49ed-9d9c-16ae9ec0a4f3.png)
